test/commands/data/export/bulk.nut.ts

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright (c) 2024, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import path from 'node:path';
+import { execCmd, TestSession } from '@salesforce/cli-plugins-testkit';
+import { expect } from 'chai';
+import { ensureNumber } from '@salesforce/ts-types';
+import { validateCsv, validateJson } from '../../../testUtil.js';
+import { DataExportBulkResult } from '../../../../src/commands/data/export/bulk.js';
+
+describe('data export bulk NUTs', () => {
+  let session: TestSession;
+  let totalAccountRecords: number | undefined;
+
+  before(async () => {
+    session = await TestSession.create({
+      scratchOrgs: [
+        {
+          config: 'config/project-scratch-def.json',
+          setDefault: true,
+        },
+      ],
+      project: { sourceDir: path.join('test', 'test-files', 'data-project') },
+      devhubAuthStrategy: 'AUTO',
+    });
+
+    // TODO: make this use the new `data import bulk` command when its available
```

Comment:
Is there a work item for this TODO?  If so, please add it to the comment.  If not, we should create one to track the work for this.